chart/values.yaml

```diff
@@ -22,6 +22,8 @@ distributor:
   config:
     queueGroup:
       enabled: true                          # Enable connection via Nats queue group to support exactly-once message processing
+  containerSecurityContext:                  # Settings for the default security context of the distributor
```

Comment:
maybe instead of having the extra `overwrite` key that will be mixed with the security context keys, we may create a dedicated key `containerSecurityContextOverride` in which people can specify the extra keys they want (at this point I dunno if we want to go for full replacement or just partial override of keys, what do you think @christian-kreuzberger-dtx ? )